behat/symfony2-extension/2.1/bin/behat

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Behat.
```

Comment:
Replacing the Behat test runner by your own script is a bad idea, as it means we will then have to keep it in sync with the one in Behat (which is not even the package for which we have a recipe).

so this is a big -1 from me.

If the goal is to load the .env file, a better way is to create a DotenvExtension for Behat instead, for cases where you want to use a .env file.